editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/restconfiguration/RestConfigEditor.java

```diff
@@ -0,0 +1,725 @@
+			if (event.getStructuredSelection().getFirstElement() instanceof Element) {
+				clearUI();
+				
+				Element restElement = (Element) event.getStructuredSelection().getFirstElement();
+				setSelection(new StructuredSelection(new CamelBasicModelElement(null, restElement)));
+				
+				if (restElement.getChildNodes().getLength() > 0) {
+					for (int i = 0; i < restElement.getChildNodes().getLength(); i++) {
+						Node child = restElement.getChildNodes().item(i);
+						if (child instanceof Element) {
+							Element elChild = (Element) child;
+							String verbUri = elChild.getAttribute("uri"); //$NON-NLS-1$
+							Composite operation = createVerbComposite(restOpsSection, elChild.getTagName(), verbUri);
+							operation.setData(RestConfigConstants.REST_VERB_FLAG, elChild);
+						}
+					}
+				}
+				restOpsSection.layout();
+			}
+		});
+
+		section.setClient(client);
+		
+		return client;
+	}
+	
+	private Composite createRestOperationTabSection() {
+		Section section = toolkit.createSection(form.getBody(), Section.EXPANDED | Section.TITLE_BAR | Section.DESCRIPTION);
+		section.setText("REST Operations");
+		section.setDescription("Specify the details for each REST operation.");
+		GridData gd = new GridData(SWT.FILL,SWT.FILL,true,true);
+		section.setLayoutData(gd);
+		section.setLayout(new GridLayout(2, false));
+
+		ToolBar toolbar = createToolbar(section);
+		section.setTextClient(toolbar);
+
+		Composite client=toolkit.createComposite(section,SWT.BORDER);
+		client.setLayout(new GridLayout(1, false));
+		client.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
+		client.setLayoutData(new GridData(SWT.FILL,SWT.FILL,true,true));
+		
+		section.setClient(client);
+		
+		return client;
+	}
+
+	class RestLabelProvider extends LabelProvider {
+
+		@Override
+		public String getText(Object element) {
+			if (element instanceof Element) {
+				Element restElement = (Element) element;
+				if (getAttrValue(restElement, "path") != null) {
+					return getAttrValue(restElement, "path");
+				}
+			}
+			return super.getText(element);
+		}
+		
+	}
+
+	private Composite createRestConfigurationTabSection() {
+		Section section = toolkit.createSection(tabFolder, Section.EXPANDED | Section.TITLE_BAR | Section.DESCRIPTION);
+		section.setText(RestConfigConstants.REST_CONFIGURATION_TAG.toUpperCase());
+		section.setDescription("Specify the details of the REST Configuration.");
+		GridData gd = new GridData(SWT.FILL,SWT.FILL,true,false);
+		section.setLayoutData(gd);
+		section.setLayout(new GridLayout(2, false));
+
+		ToolBar toolbar = createToolbar(section);
+		section.setTextClient(toolbar);
+
+		Composite client=toolkit.createComposite(section,SWT.NONE);
+		client.setLayout(new GridLayout(2, false));
+		client.setBackground(Display.getCurrent().getSystemColor(SWT.COLOR_WHITE));
+		
+		toolkit.createLabel(client, "Component:");
+		componentCombo = new Combo(client, SWT.DROP_DOWN);
+		componentCombo.add("netty-http");
```

Comment:
I think we should traverse the connectors and look for category / tag "Rest" or similar instead of hardcoding values.